### ISSUE TITLE
classification task

### ISSUE THREAD

**Issue by ReySadeghi** [UKPLab/sentence-transformers#704]
Hi,
I have a question from classification task, actually I fine-tuned both Bi-encoder and cross-encoder for binary classification task,
but in term of using, the model gives me a number between 0 and 1 that shows similarity rate, but according to the model that it has a classification layer on top of the model, it should results just the label.
could you please inform me that I can get the label in output or not?

thanks

**Comment by nreimers** [comment]
The Bi-Encoder will not save the classification layer (if used with softmax loss): Bi-Encoder purpose is to be used for computing embeddings, not for classification.

Cross-Encoder with num_labels=1 return a score 0 to 1, indicating the probability for label=1. So you define a threshold and tret e.g. all scores above 0.5 as label 1 and all below as label 0

**Comment by ReySadeghi** [comment]
what about cross-encoder with num_labels=2 ? 
it gives me an array of 2, from that how can I result my label?

the cross-encoder doesn't save the classification layer? because I want the final model could find the best threshold and at the end give me the label not a score.

**Comment by nreimers** [comment]
The cross encoder has a classification layer.

If you use num_labels = 1 and a threshold 0.5 or num_labels=2 and take the argmax does not matter, it is the same.
